### PR TITLE
Add guide on translating with Crowdin

### PR DESCRIPTION
**Pull request by melissawm**

### CODE REVIEW

**Comment by melissawm** [comment]
Some extra points we could add:
- How to suggest a new language
- How to engage with the project beyond crowdin (recommend going to project's communication channels etc)
- explicitly link to faq?

File: content/translate.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: How to translate content using Crowdin
+draft: false
+---
+
+If you want to start translating content for Scientific Python projects, you can use Crowdin to help you manage the translation process. You can see which projects are currently set up with Crowdin in our [GitHub project board](https://github.com/orgs/Scientific-Python-Translations/projects/1).
+
+You can also see [a similar guide in video format on YouTube]().
```
Comment (steppi) [inline content/translate.md]: I guess we should wait to merge this PR until after the video is completed? I take a look and provide feedback shortly.

File: content/translate.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### 1. Create Crowdin profile
+
+First, navigate to https://scientific-python.crowdin.com/ - note you will be logging in/signing up to a Crowdin Enterprise account.
```
Comment (steppi) [inline content/translate.md]: Kind of a nitpick, but I think this reads better if the prepositions match
```suggestion
First, navigate to https://scientific-python.crowdin.com/ - note you will be logging in to or signing up for a Crowdin Enterprise account.
```

**Review by steppi — COMMENTED** [review]
This is great @melissawm! I'll take a look at the tutorial video now and try to give some feedback

File: content/translate.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+First, navigate to https://scientific-python.crowdin.com/ - note you will be logging in to or signing up for a Crowdin Enterprise account.
+
+Create a username and password, or log in with your credentials (if you already have a GitHub account, you can use that to log in).
+
+<img alt="Crowdin login page showing the options of creating a new username and password, and logging in with multiple services such as Google, Facebook, GitHub and Twitter." src="../images/1-crowdin.png" width=800/>
+
+### 2. Join the Scientific Python Discord server and ask to be added as a translator
+
+Join the [Scientific Python Discord server](https://discord.gg/vur45CbwMz) and ask to be added as a translator in the `#translation` channel. This will give you access to the projects that are currently being translated.
```
Comment (melissawm) [inline content/translate.md:20]: Just added this step as previously discussed.

**Comment by melissawm** [comment]
This should be ready and should be merged before the blog post is published on the Scientific Python blog.

**Review by steppi — APPROVED** [review]
Looks good. Thanks @melissawm!

**Comment by rgommers** [comment]
@steppi this somehow didn't trigger a deployment, I don't see the new page on https://scientific-python-translations.github.io nor a deployment action running at https://github.com/Scientific-Python-Translations/scientific-python-translations.github.io/actions

**Comment by rgommers** [comment]
Ah sorry, that's gh-4. (reading things in sequence)

**Pull request merged**